changelog.d/9506.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug introduced in v1.25.0: `ValueError: Attempted to iterate a RoomID` when attempting to use `/_synapse/admin/join/` with a room alias.
```

Comment:
```suggestion
Fix a bug introduced in v1.25.0 where `/_synapse/admin/join/` would fail when given a room alias.
```

As that's probably clearer what the effect is?